tests/execution_engine/test_sqlalchemy_batch_data.py

```diff
@@ -149,3 +151,30 @@ def test_instantiation_with_unknown_dialect(sqlite_view_engine):
     )
 
     assert batch_data.dialect == GESqlDialect.OTHER
+
+
+@pytest.mark.unit
+def test_instantiation_with_temp_table_schema():
```

Comment:
I'm no Python testing wizard, so I'm happy to adjust this. It's not particularly robust -- I wanted to get the PR out first. 🙏

Reply:
@BobbyRyterski thank you very much for your contribution (and your patience in the review process).  I think this test is an elegant way to test the functionality that you added, with a positive and negative test. LGTM, and looking forward to having it be a part of the codebase (and upcoming release). Thank you again for your contribution